src/settings.h

```diff
@@ -323,6 +359,11 @@ class Settings
     static std::wstring ms_appVersion;
     static std::wstring ms_appBuildVersion;
     static std::string  ms_DSAPubKey;
+    
+    static void *ms_customConfigData;
+    static win_sparkle_config_read_t ms_currentConfigRead;
```

Comment:
I don't think this needs to say "current" in it — it's not like it's something that is supposed to change during lifetime;   OTOH it would make sense to indicate it's a function, because it's not entirely obvious when reading the code elsewhere, so `ms_configReadFunc, …, m_configCustomData`.